AuthenticationExample/AuthenticationExample/AuthenticationApp.swift

```diff
@@ -18,19 +18,9 @@ import ArcGIS
 
 @main
 struct AuthenticationApp: App {
-    @ObservedObject var authenticator: Authenticator
-    @State var isSettingUp = true
+    @StateObject private var authenticator = Authenticator()
     
-    init() {
-        // Create an authenticator.
-        authenticator = Authenticator(
-            // If you want to use OAuth, uncomment this code:
-            //oAuthUserConfigurations: [.arcgisDotCom]
-        )
-        // Sets authenticator as ArcGIS and Network challenge handlers to handle authentication
-        // challenges.
-        ArcGISEnvironment.authenticationManager.handleChallenges(using: authenticator)
```

Comment:
Now that `authenticator` is a `StateObject` object this causes a purple runtime warning. Instead, I've moved into the task below.